README.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+
+<html lang="en-US">
+    <head>
+        <title>README file</title>
+
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link href="@css/documentation_style.css" rel="stylesheet" type="text/css">
+    </head>
+
+    <body>
+      <p><a name="readme-top"></a></p>
+      
+      <!--
+      !!! IMPORTANT !!!
+      This README is an example of how you could professionally present your codebase. 
+      Writing documentation is a crucial part of your work as a professional software developer and cannot be ignored. 
+      
+      You should modify this file to match your project and remove sections that don't apply.
+      
+      REQUIRED SECTIONS:
+      - Table of Contents
+      - About the Project
+        - Built With
+        - Live Demo
+      - Getting Started
+      - Authors
+      - Future Features
+      - Contributing
+      - Show your support
+      - Acknowledgements
+      - License
+      
+      OPTIONAL SECTIONS:
+      - FAQ
+      
+      After you're finished please remove all the comments and instructions!
+      
+      For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
+      -->
+      
+      <div align="center">
+        <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+        <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] The logo you have is not visible, so i could not verify if its the origial one from the template. If it is, kindly remove it and customise it, or remove the section completely 👍